flink/flink1.10-shims/src/main/java/org/apache/zeppelin/flink/Flink110Shims.java

```diff
@@ -319,4 +319,22 @@ public Map extractTableConfigOptions() {
     }
     return configOptions;
   }
+
+  @Override
+  public String[] row2String(Object row, Object table, Object tableConfig) {
+    return rowToString((Row) row);
+  }
```

Comment:
Wouldn't it be better to declare the parameter's type as a more specific type than `Object`?
And it would be better to use either `row2String` or `rowToString` consistently.

Reply:
Thanks for the review @cuspymd I have rename method name to be rowToString. Regarding the parameter type, it is not possible to use the specific type because the api inconsistent across different flink versions, so I have to use type Object and let each version of `FlinkShims` to do the type casting

Reply:
Looking at the current implementation, the `rowToString()` function is only called with specific parameter types in the `Abstract StreamSqlJob::table2String()` function. I don't quite understand how different types can be passed.

Reply:
@cuspymd The reason is that module `flink-shims` doesn't depend on any specific version of flink api,  the flink dependency is specified in each version of flink-shims, so we can not use flink api in `flink-shims`